### PR TITLE
Fix #5568: rename cloudmc

### LINKED ISSUE #5568: Rename cloudmc to openmc

### PR DESCRIPTION
**Pull request by gurhar1133**

### CODE REVIEW

**Comment by gurhar1133** [comment]
once this is merged i can address: https://github.com/radiasoft/sirepo/pull/6824

File: sirepo/feature_config.py [hunk 1]
```diff
@@ -135,11 +135,11 @@ def _test(msg):
     _cfg = pkconfig.init(
         # No secrets should be stored here (see sirepo.job.agent_env)
         api_modules=((), set, "optional api modules, e.g. status"),
-        cloudmc=dict(
+        openmc=dict(
```
[the remark this answers is not in the document]
Comment (gurhar1133) [reply sirepo/feature_config.py]: @moellep what do you think?

[the remark this answers is not in the document]
Comment (moellep) [reply sirepo/feature_config.py]: Yes, we'll need this so people's cloudmc directory gets renamed.

**Comment by gurhar1133** [comment]
can you have a look @moellep please

**Comment by gurhar1133** [comment]
@moellep ping

File: sirepo/db_upgrade.py [hunk 2]
```diff
@@ -79,6 +79,11 @@ def _20240322_remove_github_auth(qcall):
     qcall.auth_db.drop_table("auth_github_user_t")
 
 
+def _20240507_cloudmc_to_openmc(qcall):
+    for u in qcall.auth_db.all_uids():
+        _migrate_sim_type("cloudmc", "openmc", qcall, u)
+
```
Comment (moellep) [inline sirepo/db_upgrade.py:85]: This line is failing for me with an error ```AttributeError: 'API' object has no attribute 'cookie'```.

I was on the master branch, created some cloudmc simulations, then checked out this branch and started ```sirepo service http``` and it failed when it was looking up the simulation_dir() during the db upgrade.

**Comment by gurhar1133** [comment]
Another look @moellep think i fixed it

**Review by moellep — CHANGES_REQUESTED** [review]
I find my local simulations aren't getting copied over correctly. If I have a non-example simulation, and access /openmc after the upgrade has been run, I get this server error: 
```unsupported exception=<class 'AssertionError'> msg=invalid simulation type=cloudmc```

When I look at the simulation sirepo-data.json files in run/user/*/openmc, only the example files have the correct simulationType set to "openmc". The user simulations remain with simulationType "cloudmc".

To reproduce:
- open the master branch
- delete the entry in auth.db db_upgrade_t using sqlite
- open cloudmc and create a copy of the Aurora sim
- switch to the 5568-rename-cloudmc branch
- start the server and go to /openmc

**Comment by gurhar1133** [comment]
@moellep I think you have to un block to auto-merge

**Review by moellep — APPROVED** [review]

**Pull request merged**